docs/tofu.logging.misc.md

```diff
@@ -0,0 +1,32 @@
+---
+id: tofu.logging.misc
```

Comment:
I don't think we need to broaden this to misc. My initial intention was to create a one-page-one-feature documentation, so let's name this page `tofu.logging.annotation` or something like that